core/src/test/java/org/opensearch/sql/expression/datetime/TimeDiffTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.expression.datetime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalTime;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.opensearch.sql.data.model.ExprTimeValue;
+import org.opensearch.sql.expression.DSL;
+
+public class TimeDiffTest extends DateTimeTestBase {
+
+  private static Stream<Arguments> getTestData() {
+    return Stream.of(
+        Arguments.of(LocalTime.of(12, 42), LocalTime.of(7, 40), LocalTime.of(5, 2)),
+        Arguments.of(LocalTime.of(7, 40), LocalTime.of(12, 42), LocalTime.of(18, 58)),
+        Arguments.of(LocalTime.of(7, 40), LocalTime.of(7, 40), LocalTime.of(0, 0)),
+        Arguments.of(LocalTime.MAX, LocalTime.MIN, LocalTime.MAX)
+    );
+  }
+
+  /**
+   * Test `TIME_DIFF` function with different data.
+   * @param arg1 First argument.
+   * @param arg2 Second argument.
+   * @param expectedResult Expected result.
+   */
+  @ParameterizedTest
+  @MethodSource("getTestData")
+  public void try_different_data(LocalTime arg1, LocalTime arg2, LocalTime expectedResult) {
```

Comment:
Just a note, but in JUnit expected result is usually the first argument.

Reply:
Yes, it is. Do you insist me to update all PRs to fix that?

Reply:
No, but it'd  help with readability if it was so.